src/Contracts/ActivatorInterface.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Nwidart\Modules\Contracts;
+
+use Nwidart\Modules\Module;
+
+interface ActivatorInterface
+{
+    /**
+     * Enables a module
+     *
+     * @param Module $module
+     */
+    public function enable(Module $module);
+
+    /**
+     * Disables a module
+     *
+     * @param Module $module
+     */
+    public function disable(Module $module);
+
+    /**
+     * Determine whether the given status same with a module status.
+     *
+     * @param Module $module
+     * @param $status
+     *
+     * @return bool
+     */
+    public function isStatus(Module $module, $status);
+
+    /**
+     * Set active state for a module.
+     *
+     * @param Module $module
+     * @param $active
+     */
+    public function setActive(Module $module, $active);
+
+    /**
+     * Sets a module status by its name
+     * 
+     * @param  string $name
+     * @param  $active
+     */
+    public function setActiveByName(string $name, $active);
+
+    /**
+     * Deletes a module activation status
+     * 
+     * @param  Module $module
+     */
+    public function delete(Module $module);
```

Comment:
There's delete and disable? What's the difference?

Reply:
This was only to make up for the delete method of the Module class. I assumed it's just a helper for developers or tests since I don't see any reference of it (?).
It basically deletes any activation status for that module (active or not)

Reply:
I see ok, that makes sense. I believe the delete method on the module class is there by request from the community, there could a future delete command that uses that too actually.